types/office-js/index.d.ts

```diff
@@ -11233,6 +11229,19 @@ declare namespace Office {
          * The seriesId property returns null for items that do not have parent items such as single appointments, series items, or meeting requests 
          * and returns undefined for any other items that are not meeting requests.
          * 
+         * [Api set: Mailbox 1.7]
+         * 
+         * @remarks
+         * 
+         * <table><tr><td>{@link https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions | Minimum permission level}</td><td>ReadItem</td></tr>
+         * 
+         * <tr><td>{@link https://docs.microsoft.com/outlook/add-ins/#extension-points | Applicable Outlook mode}</td><td>Compose or read</td></tr></table>
+         */
+        seriesId: string;
+
+        /**
+         * Gets the sharedProperties of an appointment or message item.
```

Comment:
Should "sharedProperties" be "SharedProperties"? Maybe the description could explain the object a bit too.